projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -19,13 +19,15 @@
+    "cy:run:punchout": "cypress run --config-file cypress.config.ci.ts --env PUNCHOUT=true --spec \"cypress/e2e/b2b/regression/punchout/*.e2e.cy.ts\"",
     "cy:run:ci": "cypress run --config-file cypress.config.ci.ts --record --key $CYPRESS_KEY --tag \"b2c,all\" --parallel --ci-build-id $BUILD_NUMBER --group B2C --spec \"cypress/e2e/!(vendor|b2b|ssr|cx_ccv2|cx_mcs)/**/*.e2e.cy.ts,!cypress/e2e/**/*.a11y-e2e.cy.ts,!cypress/e2e/**/my-account-v2*.e2e.cy.ts\" --reporter junit --reporter-options mochaFile=results/e2e-test-b2c-[hash].xml",
     "cy:run:ci:core": "cypress run --config-file cypress.config.ci.ts --record --key $CYPRESS_KEY --tag \"b2c,core\" --parallel --ci-build-id $BUILD_NUMBER --group B2C --spec \"cypress/e2e/!(vendor|b2b|ssr|cx_ccv2|cx_mcs)/**/*.core-e2e.cy.ts\"",
     "cy:run:ci:a11y": "cypress run --config-file cypress.config.ci.ts --record --key $CYPRESS_KEY --tag \"a11y,all\" --parallel --ci-build-id $BUILD_NUMBER --group A11Y --spec \"cypress/e2e/**/*.a11y-e2e.cy.ts\" --reporter junit --reporter-options mochaFile=results/e2e-test-a11y-[hash].xml",
     "cy:run:ci:b2b": "cypress run --config-file cypress.config.ci.ts --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --record --key $CYPRESS_KEY --tag \"b2b,all\" --parallel --ci-build-id $BUILD_NUMBER --group B2B --spec \"cypress/e2e/b2b/regression/**/*e2e.cy.ts\" --reporter junit --reporter-options mochaFile=results/e2e-test-b2b-[hash].xml",
     "cy:run:ci:core:b2b": "cypress run --config-file cypress.config.ci.ts --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --record --key $CYPRESS_KEY --tag \"b2b,core\" --parallel --ci-build-id $BUILD_NUMBER --group B2B --spec \"cypress/e2e/b2b/regression/**/*.core-e2e.cy.ts\"",
     "cy:run:ci:ccv2": "cypress run --config-file cypress.config.ci.ts --config baseUrl=$ENDPOINT_URL_PUBLIC_SPARTACUS --reporter junit --reporter-options mochaFile=results/spartacus-test-results-[hash].xml --env API_URL=$ENDPOINT_URL_PUBLIC_API --spec $E2ES_TO_RUN",
     "cy:run:ci:ccv2-b2b": "cypress run --config-file cypress.config.ci.ts --config baseUrl=$ENDPOINT_URL_PUBLIC_SPARTACUS --reporter junit --reporter-options mochaFile=results/spartacus-test-results-[hash].xml --env API_URL=$ENDPOINT_URL_PUBLIC_API,BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --spec \"cypress/e2e/cx_ccv2/regression/b2b/**/*.e2e.cy.ts\"",
+    "cy:run:ci:punchout": "cypress run --config-file cypress.config.ci.ts --env PUNCHOUT=true --group punchout --spec \"cypress/e2e/b2b/regression/punchout/*.e2e.cy.ts\"",
```

Comment:
I think it needs to be in vendor section so
cy:run:ci:vendor:punchout
and the cy.ts files in 
spartacus/projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/vendor/punchout
let me confirm with Kate